src/DiagnosticsProvider.php

```diff
@@ -84,6 +84,45 @@ public static function checkDiagnostics($node) {
                     }
                 }
             }
+            else if ($node instanceof Node\Statement\BreakOrContinueStatement) {
+                if ($node->breakoutLevel === null) {
+                    return null;
+                }
+
+                $breakoutLevel = $node->breakoutLevel;
+                while ($breakoutLevel instanceof Node\Expression\ParenthesizedExpression) {
+                    $breakoutLevel = $breakoutLevel->expression;
+                }
+
+                if (
+                    $breakoutLevel instanceof Node\Expression\NumericLiteral
+                    && \in_array($breakoutLevel->children->kind, [
+                        TokenKind::BinaryLiteralToken,
+                        TokenKind::DecimalLiteralToken,
+                        TokenKind::HexadecimalLiteralToken,
+                        TokenKind::OctalLiteralToken,
+                        TokenKind::IntegerLiteralToken
+                    ])
+                    && \intval($breakoutLevel->getText()) > 0
```

Comment:
why not just cast to an int?

Reply:
I don't think that works for anything but decimal numbers